extensions/google-maps-search/package.json

```diff
@@ -2,91 +2,123 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "google-maps-search",
   "title": "Google Maps Search",
-  "description": "Quickly open Google Maps directions, for example from your current location to your home.",
+  "description": "Quickly search Google Maps or get directions.",
   "icon": "google-maps-icon.png",
   "author": "ratoru",
   "categories": [
     "Web"
   ],
   "contributors": [
-    "lin"
+    "lin",
+    "chrismessina"
+  ],
+  "keywords": [
+    "directions",
+    "navigation"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "travelHome",
-      "title": "Travel Home",
+      "name": "getMeHome",
+      "title": "Get Me Home",
       "subtitle": "Google Maps",
       "description": "Open Google Maps with directions from your current location to your home.",
       "mode": "no-view"
     },
     {
-      "name": "travelTo",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
Done!